contrib/localnet/orchestrator/smoketest/test_deposit_eth.go

```diff
@@ -9,15 +9,16 @@ import (
 	"math/big"
 	"time"
 
-	ethcommon "github.com/ethereum/go-ethereum/common"
-	"github.com/zeta-chain/zetacore/x/crosschain/types"
-	"github.com/zeta-chain/zetacore/zetaclient"
-
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	ethcommon "github.com/ethereum/go-ethereum/common"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/crypto"
 	zrc20 "github.com/zeta-chain/protocol-contracts/pkg/contracts/zevm/zrc20.sol"
 	"github.com/zeta-chain/zetacore/common"
+	ethereum2 "github.com/zeta-chain/zetacore/common/ethereum"
```

Comment:
```suggestion
	ethereum "github.com/zeta-chain/zetacore/common/ethereum"
```
It doesn't seem to be necessary as the other import is `ethcommon`